src/swap/defi/thorchain/swapkit.ts

```diff
@@ -231,11 +234,9 @@ export function makeSwapKitPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
 
     if (exchangeInfo != null) {
       const { swapkit } = exchangeInfo.swap.plugins
+      affiliateFeeBasis = swapkit.affiliateFeeBasis ?? affiliateFeeBasis
```

Comment:
why are we allowing the info server to override the fee?

Reply:
The thorchain swap plugin was written to do this and since the two plugins used to share an exchangeInfo cleaner it seemed like it was supposed to be here, too.